src/middlewared/middlewared/plugins/vm.py

```diff
@@ -1634,6 +1654,69 @@ def initialize_vms(self, timeout=10):
                 # Whatever happens, we don't want middlewared not booting
                 self.vms[vm_data['name']] = VMSupervisor(vm_data, self.libvirt_connection, self.middleware)
 
+    @private
+    def setup_libvirt_events(self):
```

Comment:
I am continuing to push my idea of making our files smaller (or at least not increasing their size). Please put this to separate file (for existing modules we do it like (`vm_/events.py`)